RStudioProxy/service/instance.js

```diff
@@ -0,0 +1,41 @@
+const { redis } = require('../infrastructure')
+
```

Comment:
due to the new agreement the calls to redis now are going to be moved to the PoolService
and a new InstanceModel is going to be created